File: flutter/test_app/lib/data/supabase/model/DTOs/korisnik_dto.dart
class KorisnikDto {
  final String idKorisnika;
  final String imeKorisnika;
  final String prezimeKorisnika;

  KorisnikDto({
    required this.idKorisnika,
    required this.imeKorisnika,
    required this.prezimeKorisnika,
  });

  factory KorisnikDto.fromJson(Map<String, dynamic> json) {
    return KorisnikDto(
      idKorisnika: json['IdKorisnika'] as String,
      imeKorisnika: json['ImeKorisnika'] as String,
      prezimeKorisnika: json['PrezimeKorisnika'] as String,
    );
  }
}

File: kotlin_jetpack_compose/app/src/main/java/com/example/testapp/data/supabase/model/DTOs/KorisnikDTO.kt
package com.example.testapp.data.supabase.model.DTOs

import kotlinx.serialization.Serializable
import kotlinx.serialization.SerialName

@Serializable
data class KorisnikDTO(
    @SerialName("IdKorisnika")
    val idKorisnika: String,
    @SerialName("ImeKorisnika")
    val imeKorisnika: String,
    @SerialName("PrezimeKorisnika")
    val prezimeKorisnika: String
)